### DISCUSSION TITLE
Maximum files per directory

### DISCUSSION THREAD

**Discussion by tunes0710** [filebrowser/filebrowser#1284] [Q&A] [answered]
Hiya,
I have filebrowser installed on a box that is linked to a mounted google drive.
When I go in to a folder on the drive that has lots of files I get the "Something really went wrong." message.
Is there a setting to increase the max number of files per directory that it supports?

Cheers :)

**Comment by ramiresviana** [comment, marked as answer]
Filebrowser was no such control of max number of files per directory, it uses standard system syscalls to access filesystem. Supporting cloud storages is out of the scope of this project.
